### ISSUE TITLE
Improve accessibility of UmpleOnline by making all key elements tabbable

### ISSUE THREAD

**Issue by TimLethbridge** [umple/umple#1811]
To improve accessibility it should be possible to tab to as many as possible of the elements of UmpleOnline.

Guidelines are here https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard

This means adding tabindex=0 to all buttons (along with onKeyDown functions to make them do the same thing they do if clicked on). For the main editing window it would lkely need special treatment with role="document".

For example the user should be able to tab and use the return key to select Download, Donate, User manual, ... E, G, S ... Save & Load (which would be opened by a return). Some items already are tabbable, but there is no onKeyDown associated with them

The code to change would be in umpleonline/umple.php and umpleonline/scripts/*
